src/polymer-project.ts

```diff
@@ -327,6 +327,24 @@ class HtmlSplitter extends Transform {
           this.push(scriptFile);
         }
 
+        for (let i = 0; i < styleTags.length; i++) {
+            let styleTag = styleTags[i];
+            let source = dom5.getTextContent(styleTag);
+            let extension = 'css';
+            let childFilename = `${osPath.basename(filePath)}_style_${i}.${extension}`;
+            let childPath = osPath.join(osPath.dirname(filePath), childFilename);
+            styleTag.childNodes = [];
+            dom5.setAttribute(styleTag, 'src', childFilename);
```

Comment:
This will make a tag like: `<style src="x.css"></style>`, when it needs to be `<link rel="stylesheet" href="x.css">`